CHANGELOG-2.0.md

```diff
@@ -1,3 +1,9 @@
+# Changelog since v2.0.0
+
+## Breaking Changes
+
+- Cherry pick PR #240: Fix go module requirements for semantic versioning. Update package path to v2. Vendoring with dep depends on https://github.com/golang/dep/pull/1963 or the workaround described in v2/README.md. ([#243](https://github.com/kubernetes-csi/external-snapshotter/pull/243), [@alex1989hu](https://github.com/alex1989hu))
```

Comment:
@jsafrane can you check if attacher also has similar issue?

Reply:
Maybe. I admit I still don't fully understand the issue even after reading https://github.com/kubernetes-csi/external-snapshotter/issues/239

Reply:
Maybe. I admit I still don't fully understand the issue even after reading #239

Reply:
If someone needs to do a "go get -v github.com/kubernetes-csi/external-attacher@v2.1.0", that will probably fail with the similar error because there's no v2 in the package path.

Reply:
If you try to import a go module that's tagged with >= 2.0, the packages need to be in v2 directory. But some workarounds need to be done to still make it work with dep

Reply:
ack, filled https://github.com/kubernetes-csi/external-attacher/issues/208

Reply:
After cutting the v2.0.1 release, I'm still getting the same errors.  Need to figure out what else needs to be fixed.

Reply:
Kindly ask you to see my comments:
- [issuecomment-577791852](https://github.com/kubernetes-csi/external-snapshotter/issues/239#issuecomment-577791852)
- [issuecomment-577852778](https://github.com/kubernetes-csi/external-snapshotter/issues/239#issuecomment-577852778)

Reply:
Thanks @alex1989hu.  I'll give it a try.

Reply:
Verified "go get -u github.com/kubernetes-csi/external-snapshotter/v2@v2.0.1" works for me now.
Also tested the manual steps to generate code and that works too.